main/src/preprocessing/tokenizer/base_tokenizer.rs

```diff
@@ -328,7 +324,7 @@ pub trait Tokenizer<T: Vocab> {
             .replace(" ,", ",")
             .replace(" ' ", "'")
             .replace(" n't", "n't")
-            .replace(" 'm", "'m")
+            .replace(" 'm'", "'m")
```

Comment:
Ah? So it wasn't a typo after all? My rust-be tests are green now with this re-commit!

Reply:
It was a typo. The tests for `rust-bert` probably need to be updated. I am about to push some changes that re-re-fixes the typo you found in a previous PR but got lost. Sorry that is what I meant.